login-module/src/index.js

```diff
@@ -1,19 +1,18 @@
-import constants from './constants';
-import { WebAuth } from 'auth0-js';
-import Quiz from './quiz';
-import Enrollment from './enrollment';
-import Certificate from './certificate';
-import misc from './misc';
-
-window.GraphAcademyLogin = class GraphAcademyLogin {
+import Auth0 from 'auth0-js';
```

Comment:
Node.js cannot resolve `import { WebAuth } from 'auth0-js'` because `WebAuth` is not exported using ESM :disappointed: